lago/cmd.py

```diff
@@ -154,24 +162,17 @@ def do_init(
                 template_repo_path
             )
         else:
-            try:
-                repo_name = (
-                    template_repo_name
-                    or lago.config.get('template_default_repo')
```

Comment:
Careful! this might break old configs (as it's not using the template_default_repo value anymore)